neqo-http3/src/connection.rs

```diff
@@ -1256,9 +1258,13 @@ mod tests {
         let _ = connect(false);
     }
 
-    fn connect_and_receive_control_stream(
-        client: bool,
-    ) -> (Http3Connection, Connection, u64, QPackEncoder) {
+    struct PeerConnection {
+        conn: Connection,
+        control_stream_id: u64,
+        encoder: QPackEncoder,
+    }
+
+    fn connect_and_receive_control_stream(client: bool) -> (Http3Connection, PeerConnection) {
```

Comment:
I wonder if this argument could be changed to an enum so that call sites can see `connect_and_receive_control_stream(Client)`.  Also, this could probably just be renamed to `connect()`.